doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -126,6 +126,7 @@ nRF9160: Asset Tracker v2
 
   * The application now uses the new LwM2M location assistance objects through the :ref:`lib_lwm2m_location_assistance` library.
   * Added handling for the new data receive events in the :ref:`lib_nrf_cloud` library.
+  * The application now uses passive mode as the default mode for Thingy:91 builds.
```

Comment:
dont we need to mention the change about the (in)activity thresholds too?

Reply:
I think changing the default values of the thresholds is a minor change. Since they are so application-specific, customers will tweak them anyways.